js/maps.js

```diff
@@ -1,3 +1,5 @@
+import { queryParams } from './queryParams.js'
```

Comment:
I've been avoiding `import` because it may not work in the browser.

Reply:
Ignore that, [MDN says it has good support now](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#Browser_compatibility).

Reply:
as the comment about good support on this import statement? It looked like it had ok support when i went this way - can remove if needed.